modules/data-entry/src/main/resources/SLING-INF/content/oak%3Aindex/forms.json

```
@@ -49,6 +49,20 @@
                     "propertyIndex": true,
                     "nodeScopeIndex": false,
                     "jcr:primaryType": "nt:unstructured"
+                },
+                "lastModified": {
+                    "name": "jcr:lastModified",
+                    "ordered": true,
+                    "propertyIndex": true,
+                    "nodeScopeIndex": false,
+                    "jcr:primaryType": "nt:unstructured"
+                },
+                "statusFlags": {
+                    "name": "statusFlags",
+                    "propertyIndex": true,
+                    "nodeScopeIndex": true,
+                    "analyzed": true,
```

This is not free text, but a predefined set of possible values, so it should not be analyzed, or part of the node's text content.

```suggestion
                    "nodeScopeIndex": false,
                    "analyzed": false,
```